data/settings.js

```diff
@@ -3509,6 +3509,31 @@ many connections to the doveadm service.
 If \`0\`, commands are run directly in the same process.`
 	},
 
+	dovecot_config_version: {
+		values: setting_types.STRING,
+		text: `
+Dovecot configuration version. This must be the first setting in the
+configuration file. It specifies the configuration syntax, the used setting
+names and the expected default values.
+
+When there are default configuration changes in newer Dovecot versions, the
+existing installations will continue to work the same as before with the same
+default settings until this version number is increased. If there are other
+configuration changes, the old configuration will either keep working or there
+will be a clear failure at startup.`
+	},
+
+	dovecot_file_format_version: {
```

Comment:
Should be dovecot_storage_version right?

Reply:
Right, forgot to update documentation.